ui/lib/pki/addon/components/page/pki-tidy-status.hbs

```diff
@@ -7,10 +7,10 @@
   <ToolbarActions>
     <div class="toolbar-separator"></div>
     {{#if @autoTidyConfig.enabled}}
-      <ToolbarLink @route="tidy.auto" data-test-pki-auto-tidy-config>
+      <ToolbarLink @route="tidy.auto" @models={{array @autoTidyConfig.id}} data-test-pki-auto-tidy-config>
```

Comment:
why don't we need the backend here?

Reply:
Oh nvm, tidy config is at the top level (for the PKI mount)

Reply:
We do! Good catch 🕵️‍♀️

Reply:
Oh wait -- so for this one, `autoTidyConfig`'s ID _is_ the backend, since there's only one auto tidy per PKI mount. The reason it's passed as `models` instead of `model` is because `ToolbarLink` only accepts `models` 🙃